lib/git.js

```diff
@@ -63,7 +63,7 @@ const commitAndPush = async function(path, branch, gitRoot, logger) {
     logger.debug(`Committing assets ${path}`)
 
     git.add(path)
-    await git.commit('Compiling assets', [], { '--signoff': null }, function(err, result) {
+    await git.commit('Compiling assets', [], { '--signoff': null, '--amend': null }, function(err, result) {
```

Comment:
Unrelated, but bothers me :)

```suggestion
    await git.commit('Compile assets', [], { '--signoff': null, '--amend': null }, function(err, result) {
```

Ref https://chris.beams.io/posts/git-commit/#imperative